### PR TITLE
Ask if billing, otherwise exit

### PR DESCRIPTION
**Pull request by carise**

### CODE REVIEW

File: scripts/gcp_setup/environment/gcloud_env.py [hunk 1]
```diff
@@ -304,6 +305,33 @@ def get_project(self):
...
+                print('Billing IS enabled.')
+            else:
+                self._billing_not_enabled()
+        except ValueError:
+            self._billing_not_enabled()
+
+    def _billing_not_enabled(self):
+        """Print message and exit."""
+        print('\nIt seems that billing is not enabled for your project. '
+              'You can check whether billing has been enabled in the '
```
Comment (mirons-google) [inline scripts/gcp_setup/environment/gcloud_env.py]: nit: A message about re-running once billing is enabled would be nice.

File: scripts/docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```
Comment (mirons-google) [inline scripts/docker/docker-compose.yml]: Was this intended to be part of this PR?

File: scripts/gcp_setup/environment/gcloud_env.py (hunk 1 above)
Comment (carise) [reply scripts/gcp_setup/environment/gcloud_env.py]: Done

**Comment by carise** [comment]
Thanks!

**Pull request merged**